pkgs/development/libraries/tracker/default.nix

```diff
@@ -104,6 +107,8 @@ stdenv.mkDerivation rec {
     # We are using a symlink that will be overridden during installation.
     mkdir -p $out/lib
     ln -s $PWD/src/libtracker-sparql/libtracker-sparql-3.0.so $out/lib/libtracker-sparql-3.0.so.0
+  '' + lib.optionalString stdenv.isDarwin ''
```

Comment:
Shouldn't the `ln` command in line 109 also be inside an `optionalString` for non-Darwin?

Reply:
Fixed.